contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -182,7 +184,7 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
             uint40
         )
     {
```

Comment:
What about exposing `getQueueLength`? I'm currently calculating it by `getNumPendingQueueElements + getPendingQueueIndex`

Reply:
will do

Reply:
done!